shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
Hi,

I'm not sure how I can be of help without some extra information. If you do `git reset` that will change the HEAD pointer to a different commit, and if you run `git push origin master` afterward, you'll get the same effect in the remote repo. Was this what you did? These steps delete nothing, every commit is still there.

But perhaps it would help if you told us what it was you tried to achieve in the first place?